Sources/Benchmarks/AnyValueBenchmarks.swift

```diff
@@ -0,0 +1,417 @@
+// Copyright 2020 Penguin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Benchmark
+import PenguinStructures
+
+@_optimize(none)
+fileprivate func doNotOptimizeAway1(_ x: UnsafeRawPointer) {}
```

Comment:
Please add a TODO to move these utilities to swift-benchmark or some other library for re-use.

Reply:
I filed this issue already: https://github.com/google/swift-benchmark/issues/69

Reply:
Having a TODO in the code here (with a pointer to that issue) can help provide context to future readers who might not know of that issue. :-)